src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx

```diff
@@ -64,6 +64,15 @@ export const RecentTickets = () => {
     <View
       style={{backgroundColor: theme.static.background.background_1.background}}
     >
+      <ThemeText
+        type="body__secondary"
+        style={{
+          marginTop: theme.spacings.xLarge,
+          marginLeft: theme.spacings.xLarge,
+        }}
```

Comment:
Set these fields in `useStyles` hook instead. I see there are other places where this hasn't been followed in this file, but I think it is good to do it "correctly" even though.